Src/IronPython.Modules/nt.cs

```diff
@@ -817,11 +952,11 @@ public class uname_result : PythonTuple {
         /// <summary>
         /// Convert a sequence of args to a string suitable for using to spawn a process.
         /// </summary>
-        private static string ArgumentsToString(CodeContext/*!*/ context, object args) {
+        private static string ArgumentsToString(CodeContext/*!*/ context, object args, string methodname) {
             IEnumerator argsEnumerator;
             System.Text.StringBuilder sb = null;
             if (!PythonOps.TryGetEnumerator(context, args, out argsEnumerator)) {
-                throw PythonOps.TypeError("args parameter must be sequence, not {0}", DynamicHelpers.GetPythonType(args));
+                throw PythonOps.TypeError("args parameter must be sequence, not {1}", DynamicHelpers.GetPythonType(args));
```

Comment:
Not sure why this changed from `{0}` to `{1}`?

Reply:
Wow, no idea why...